app/src/main/java/com/mobileproto/hireddit/hireddit/visuals/SpeakFragment.java

```diff
@@ -143,6 +147,8 @@ public void onClick(View v)
         });
         helloReddit.setTypeface(tf);
 
+        spokenString = "why are my hands so smelly?";
+        new GetWordsAsync(spokenString, importantWords, getActivity(), commentText).execute();
```

Comment:
Wait, this is speak fragment. It should be deleted why I don't get is that in my branch right now, which it what it says it's this pull request in merging, things are different than they are in the pull request....